plugin.rb

```diff
@@ -8,20 +8,26 @@
 require 'json'
 require File.expand_path('../lib/validators/discourse_slack_enabled_setting_validator.rb', __FILE__)
 
-enabled_site_setting :slack_enabled
-
 PLUGIN_NAME = "discourse-slack-official".freeze
 
+enabled_site_setting :slack_enabled
+
 register_asset "stylesheets/slack_admin.scss"
 
 after_initialize do
 
   unless ::PluginStore.get(PLUGIN_NAME, "not_first_time")
+    id = SecureRandom.hex(16)
+    ::PluginStore.set(PLUGIN_NAME, "filter_#{id}", { category_id: '*', channel: "#general", filter: "follow", tags: [] })
+    ::PluginStore.set(PLUGIN_NAME, "_category_*_#general", id)
     ::PluginStore.set(PLUGIN_NAME, "not_first_time", true)
-    ::PluginStore.set(PLUGIN_NAME, "category_*", [{ category_id: '0', channel: "#general", filter: "follow" }])
   end
 
   module ::DiscourseSlack
+    def self.plugin_name
+      PLUGIN_NAME
```

Comment:
Yea I would consider it a bad practice since we're adding a constant unnecessarily. Using `DiscourseSlack::PLUGIN_NAME` is fine because that is how you reference a constant. 

> I preferred it to make it shorter.

By making it shorter, we added a few more constants around our code base which isn't a good tradeoff IMO.